autogtp/main.cpp

```diff
@@ -219,10 +211,7 @@ bool run_one_game(QTextStream& cerr, QString weightsname) {
             move_cmd = "genmove w\n";
         }
         /// Send genmove to the right process
-        auto proc = std::ref(first_process);
-        if (!first_to_move) {
-            proc = std::ref(second_process);
-        }
+        auto proc = std::ref(process);
```

Comment:
We no longer need to use std::ref if there's only one process. (This means the proc.get() stuff can go too)